packages/store-indexer/src/bin/sqlite-indexer.ts

```diff
@@ -28,6 +29,11 @@ const env = parseEnv(
     z.object({
       SQLITE_FILENAME: z.string().default("indexer.db"),
       SENTRY_DSN: z.string().optional(),
+      ENABLE_QUERY_ENDPOINT: z
```

Comment:
I would give this a bit stronger name
```suggestion
      ENABLE_UNSAFE_QUERY_API: z
```

Reply:
nice suggestion!